apps/backend/migrations/20230725092535-change-group-names-to-unique.js

```diff
@@ -0,0 +1,77 @@
+'use strict';
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.sequelize.query(`
+    CREATE OR REPLACE PROCEDURE update_group_names_transaction()
+    LANGUAGE plpgsql
+    AS $$
+    DECLARE
+      check_name boolean := true;
+      offSet_cnt INTEGER :=1;
```

Comment:
seems like spaces are fine here (and elsewhere) for assignment

```suggestion
      offSet_cnt INTEGER := 1;
```